packages/atomic/src/components/common/search-box/search-text-area.tsx

```diff
@@ -20,14 +20,12 @@ interface Props extends JSXBase.TextareaHTMLAttributes<HTMLTextAreaElement> {
 
 function getPopupAttributes(props: Required<Props>['popup']) {
   return {
-    role: 'combobox',
     autocomplete: 'off',
     autocapitalize: 'off',
     autocorrect: 'off',
     ...(props.activeDescendant && {
       'aria-activedescendant': props.activeDescendant,
     }),
-    'aria-expanded': `${props.hasSuggestions && props.expanded}`,
```

Comment:
Not supported by the implicit `textbox` role

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#associated_roles